### PR TITLE
Add content to AdAstra Notebooks

### PR DESCRIPTION
**Pull request by shubham3121**
## Description
Please include a summary of the change, the motivation, and any additional context that will help others understand your PR. If it closes one or more open issues, [please tag them as described here](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).

[ch-4565]

## Affected Dependencies
List any dependencies that are required for this change.

## How has this been tested?
- Describe the tests that you ran to verify your changes.
- Provide instructions so we can reproduce.
- List any relevant details for your test configuration.

## Checklist
- [x] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [x] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [x] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [x] My changes are covered by tests

### CODE REVIEW

File: notebooks/adastra/data-owners/00-data-owners-create-domain.ipynb [hunk 1]
```diff
@@ -8,21 +8,33 @@
...
+  },
   {
    "cell_type": "markdown",
    "id": "f4203778-938a-4af2-8aaa-ec126ef95cd1",
    "metadata": {},
    "source": [
     "## Option 1: Azure Marketplace\n",
-    "Create an [OpenMined PyGrid Domain](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/madhavajay1632269232059.openmined_mj_grid_domain_ubuntu_1?tab=Overview) node on the Azure Marketplace"
+    "\n",
+    "We can create an [OpenMined PyGrid Domain](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/madhavajay1632269232059.openmined_mj_grid_domain_ubuntu_1?tab=PlansAndPrice) node from the Azure Marketplace."
```
Comment (rasswanth-s) [inline notebooks/adastra/data-owners/00-data-owners-create-domain.ipynb:29]: MAYBE: When we click the link the in the azure market place, we could ensure the default version is 0.7.0 by itself, without users having to click and change.
Currently it shows 0.6.0 when we click the link

Comment (shubham3121) [reply notebooks/adastra/data-owners/00-data-owners-create-domain.ipynb:29]: Yep, I guess that's the plan when we launch a stable `0.7.0`, we will make it default.

File: notebooks/adastra/data-owners/00-data-owners-provision-ubuntu.ipynb [hunk 2]
```diff
@@ -127,67 +130,67 @@
...
    "metadata": {},
    "source": [
-    "![hagrid_runs](img/hagrid_runs.png)"
+    "HAGrid checks if all dependencies required for provisioning a Domain/Network node are installed. One can run: \n",
+    "```bash\n",
+    "$ hagrid --help\n",
+    "```\n",
+    "to see all the list of possible commands that can be run with the HAGrid cli tool.\n",
+    "\n",
+    "**Note**: We can see that *Docker* is not installed yet, but HAGrid will install it for us in the next step, when we provision our Domain node."
```
Comment (rasswanth-s) [inline notebooks/adastra/data-owners/00-data-owners-provision-ubuntu.ipynb:161]: Wonderful,Did not know that , hagrid also installs docker for us, I initiallly thought ,  it installs only Docker compose v2 if not present.

File: notebooks/adastra/data-owners/01-data-owners-start.ipynb [hunk 3]
```diff
@@ -5,26 +5,28 @@
...
   },
   {
    "cell_type": "markdown",
    "id": "0f2b049f-2237-4bd5-afad-6f530cb94435",
    "metadata": {},
    "source": [
-    "## Set Your Variables"
+    "## Setup Variables\n",
+    "\n",
+    "Before we start let's some variables that will come in handy later in the notebook."
```
Comment (rasswanth-s) [inline notebooks/adastra/data-owners/01-data-owners-start.ipynb:18]: NITPICK : store some variables.

File: notebooks/adastra/data-owners/01-data-owners-start.ipynb [hunk 4]
```diff
@@ -34,27 +36,22 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# edit the IP for the Network node into this variable\n",
+    "# Set the Ip Address of your Network node in this variable.\n",
+    "# The network node with IP `13.83.1.243` is managed and maintained by OpenMined.\n",
```
Comment (rasswanth-s) [inline notebooks/adastra/data-owners/01-data-owners-start.ipynb:40]: I think the network ip `13.83.1.243` was not working before, should we leave it in the comments?

File: notebooks/adastra/data-owners/01-data-owners-start.ipynb [hunk 5]
```diff
@@ -34,27 +36,22 @@
...
-  {
-   "cell_type": "code",
-   "execution_count": 46,
+   "execution_count": 10,
    "id": "a16e5bbe-c7e1-4154-b564-ab8988ee88e2",
    "metadata": {},
    "outputs": [],
    "source": [
+    "# Set the email and password of your Domain node.\n",
+    "# We will using the default email and password that got created during Domain creation.\n",
```
Comment (rasswanth-s) [inline notebooks/adastra/data-owners/01-data-owners-start.ipynb:52]: NITPICK: will be

File: notebooks/adastra/data-owners/01-data-owners-start.ipynb [hunk 6]
```diff
@@ -64,139 +61,186 @@
...
+   "source": [
+    "### Using the PySyft library\n",
+    "\n",
+    "Let's use the `syft` library to login in to your domain and get an authenticated client to your Domain node.\n",
+    "\n",
+    "To login into the your domain you will need the following credentials:\n",
+    "- url to the domain: Here the value in the `DOMAIN_HOST_IP` is the url to your domain.\n",
+    "- email address: We will use the default email (`ADMIN_EMAIL`) set on domain creation\n",
+    "- passwor: We will use the  default password (`ADMIN_PASSWORD`) set on domain creation\n",
+    "- port number (defaults to 80)"
```
Comment (rasswanth-s) [inline notebooks/adastra/data-owners/01-data-owners-start.ipynb:150]: NITPICK: port number -> port number:

File: notebooks/adastra/data-owners/01-data-owners-start.ipynb [hunk 7]
```diff
@@ -64,139 +61,186 @@
...
+   "metadata": {},
+   "source": [
+    "### Using the PySyft library\n",
+    "\n",
+    "Let's use the `syft` library to login in to your domain and get an authenticated client to your Domain node.\n",
+    "\n",
+    "To login into the your domain you will need the following credentials:\n",
+    "- url to the domain: Here the value in the `DOMAIN_HOST_IP` is the url to your domain.\n",
+    "- email address: We will use the default email (`ADMIN_EMAIL`) set on domain creation\n",
+    "- passwor: We will use the  default password (`ADMIN_PASSWORD`) set on domain creation\n",
```
Comment (rasswanth-s) [inline notebooks/adastra/data-owners/01-data-owners-start.ipynb:149]: NITPICK: password:

**Review by rasswanth-s — APPROVED** [review]
Great work, really liked the extensive explanation and the flow, left some comments

**Pull request merged**